### DISCUSSION TITLE
How do I get panoramic image observations?

### DISCUSSION THREAD

**Discussion by PengkunLiu** [allenai/ai2thor#783] [Q&A] [answered]
Sorry to disturb you. I would like to know is there any API for getting the panoramic image observations?

**Comment by winthos** [comment, marked as answer]
I believe you can try setting the `fieldOfView` value of the agent to a larger value to replicate a panoramic image.

[Check out the documentation for this here.](https://ai2thor.allenai.org/ithor/documentation/initialization/#initialization-fieldofview)
